packages/core/src/common/strings.ts

```diff
@@ -26,3 +26,13 @@ export function* split(s: string, splitter: string): IterableIterator<string> {
         start = end + splitter.length;
     }
 }
+
+// tslint:disable-next-line:no-any
+export function escapeInvisibleChars(prefValue: any): any {
```

Comment:
This is `strings` utility file so it is strange to have here functions that receive argument with `any` type